### ISSUE TITLE
Unable to select CSS/JS files via assets outside of Gantry-Assets

### ISSUE THREAD

**Issue by hennysmafter** [gantry/gantry5#1897]
Dear Developers,

I just asked on the chat how to pick a .js file that is located inside the default Joomla images folder and add this file to the Gantry > Page Settings > Assets

I setup the Plugin to include the Joomla Images Directory but still when opening the picker it is not there. The only place it is visible is when I want to pick a image via the image picker.

I am not sure if this is a bug or a feature request. If it is the latter then please allow us to pick from any directory. Either by having an option in the plugin that allows for it or just have it standard allowed. I added a picture that shows how I think would be a nice way to incorporate it.

Picture 1: The file picker I am talking about.
![010317-1](https://cloud.githubusercontent.com/assets/4426337/23465237/e9df828a-fe97-11e6-8352-86ff1959cce2.jpg)

Picture 2: The image that shows there is no JoomlaRoot/Images folder init.
![010317-2](https://cloud.githubusercontent.com/assets/4426337/23465257/f8a1be46-fe97-11e6-9442-fabafacee5fc.jpg)

Picture 3: I think this would be a cool idea.
![010317-3](https://cloud.githubusercontent.com/assets/4426337/23465280/0df017b6-fe98-11e6-91ae-9061c898f7f0.jpg)

So basically in the image above we have the default Gantry Assets opened fully as that is where most of the time the items come from but above that we have closed the `Joomla Root` area. In case someone needs to take a file from there they just open it go to the directory and add it. Very simple and straight forward. At least for the end user :-) I can imagine it is a bit more work then just writing this ticket :-)

Thanks for reading and looking forward to hear if this is going to be implemented.

**Comment by mahagr** [comment]
Sounds like a missing feature to me.
